tests/config/nf-test.config

```diff
@@ -33,6 +33,12 @@ profiles {
         docker.userEmulation = false
         docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
     }
+    docker_self_hosted{
+        docker.enabled = true
+        docker.userEmulation = false
```

Comment:
`docker.userEmulation` should be removed here too

Reply:
done, thanks